.github/dependabot.yml

```
@@ -25,3 +25,11 @@ updates:
       interval: "monthly"
     commit-message:
       prefix: "npm:"
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "docker"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: ".docker:"
```

Nitpick re prefix, how about just `docker`?
```suggestion
      prefix: "docker:"
```

Btw, we should add an exclusion for this to [goreleaser](https://github.com/ConduitIO/conduit/blob/3552ad5c2e03c63f23bf1e031a3984b12aa611c5/.goreleaser.yml#L41-L46).

---

I'm not exactly sure where we got that from, I assume that `.github` points towards updating the contents of the `.github` folder in the root of the repo 🤷 I wouldn't mind if that would simply be `github` tbh, but I don't think it's worth changing now.